app/views/posts/index.html.erb

```diff
@@ -1 +1,24 @@
-<h1>All posts for individual author are here </h1>
\ No newline at end of file
+<h1 class="user-name">                 <%=@user.name%><span class="posts_counter">Number of Posts: <%=@user.posts_counter%></span></h1>
+<ul>
+<%@user.posts.each_with_index do |post,index| %>
+
+  <li><h4>Post #<%=index + 1%></h4>
+  <%= post.text %>
```

Comment:
- [ ] @Lul-Abdifan Great work adding the posts index page, however, its missing a link to take the user to the post-show page, you can make the title to be the link.